blood/README.md

```diff
@@ -1,22 +1,83 @@
 # Blood Types
 
-TODO
+Determine the blood types of each member of a family. 
+
+```
+$ ./blood
+Generation 0, blood type OO
+    Generation 1, blood type AO
+        Generation 2, blood type OA
+        Generation 2, blood type BO
+    Generation 1, blood type OB
+        Generation 2, blood type AO
+        Generation 2, blood type BO
+
+```
 
 ## Background
 
-TODO
+Blood types are determined by two alleles (forms of a gene), and the possible alleles include A, B, and O. We get one allele from each parent, and the determination of which allele is random.
```

Comment:
Instead of "the determination of which allele is random", probably more precise to say something more like "each of a child's parents will randomly pass one of their two blood type alleles to their child"